README.md

```diff
@@ -1,82 +1,56 @@
-          | %%%%%%%%|_  %%_/   /%%__  %%| %%_  %%_  %%| %% /%%_____/      | %% |____  %% /%%__  %%
-          | %%__  %%  | %%    | %%  \ %%| %% \ %% \ %%| %%| %%       /%%  | %%  /%%%%%%%| %%  \__/
-          | %%  | %%  | %% /%%| %%  | %%| %% | %% | %%| %%| %%      | %%  | %% /%%__  %%| %%
-          | %%  | %%  |  %%%%/|  %%%%%%/| %% | %% | %%| %%|  %%%%%%%|  %%%%%%/|  %%%%%%%| %%
-          |__/  |__/   \___/   \______/ |__/ |__/ |__/|__/ \_______/ \______/  \_______/|__/
-
-
-          Congratulations on running your first test on Testcontainers Cloud! 🎉
-          You can now return to the website to complete your onboarding
-
---- PASS: TestWithRedis (4.38s)
-    --- PASS: TestWithRedis/Container_can_start (0.06s)
-    --- PASS: TestWithRedis/Connected_to_Testcontainers_Cloud (0.00s)
+
+████████╗███████╗███████╗████████╗ ██████╗ ██████╗ ███╗   ██╗████████╗ █████╗ ██╗███╗   ██╗███████╗██████╗ ███████╗ 
+╚══██╔══╝██╔════╝██╔════╝╚══██╔══╝██╔════╝██╔═══██╗████╗  ██║╚══██╔══╝██╔══██╗██║████╗  ██║██╔════╝██╔══██╗██╔════╝ 
+   ██║   █████╗  ███████╗   ██║   ██║     ██║   ██║██╔██╗ ██║   ██║   ███████║██║██╔██╗ ██║█████╗  ██████╔╝███████╗ 
+   ██║   ██╔══╝  ╚════██║   ██║   ██║     ██║   ██║██║╚██╗██║   ██║   ██╔══██║██║██║╚██╗██║██╔══╝  ██╔══██╗╚════██║ 
+   ██║   ███████╗███████║   ██║   ╚██████╗╚██████╔╝██║ ╚████║   ██║   ██║  ██║██║██║ ╚████║███████╗██║  ██║███████║ 
+   ╚═╝   ╚══════╝╚══════╝   ╚═╝    ╚═════╝ ╚═════╝ ╚═╝  ╚═══╝   ╚═╝   ╚═╝  ╚═╝╚═╝╚═╝  ╚═══╝╚══════╝╚═╝  ╚═╝╚══════╝ 
+  
+  
+  Congratulations on running your first test! 🎉
+  Runtime used: 
+      Testcontainers Cloud
+ 
+  You can now return to the website to complete your onboarding.
+
+2023/08/16 17:33:13 🐳 Terminating container: c2dbde862d74
+2023/08/16 17:33:13 🚫 Container terminated: c2dbde862d74
+--- PASS: TestTestcontainersCloud (2.63s)
 PASS
-ok  	github.com/AtomicJar/testcontainers-cloud-go-example	4.750s
+ok      github.com/AtomicJar/testcontainers-cloud-go-example    2.946s
 ```
 
-Agent status:
+## (optional) Use Testcontainers Desktop to easily debug the database
 
-![agent-running](./docs/active-connection.png)
+[Testcontainers Desktop](https://testcontainers.com/desktop/) helps developers with common tasks such as debugging your Testcontainers-powered dependencies. Let's practice!
 
-[tcc]: https://testcontainers.cloud/
+The tests in this project create a PostgreSQL database and populate it with sample data. You can [set a fixed port](https://newsletter.testcontainers.com/announcements/set-fixed-ports-to-easily-debug-development-services) for the `postgres` service, then [freeze containers shutdown](https://newsletter.testcontainers.com/announcements/freeze-containers-to-prevent-their-shutdown-while-you-debug) to easily connect to the database from your IDE after your tests run.
 
-[tcc-download]: https://app.testcontainers.cloud/start/download?mode=update
+See if you can inspect the database. Username: `test`. Password: `test`.
```

Comment:
See https://github.com/AtomicJar/testcontainers-cloud-go-example/pull/4/files#diff-79ce3229c13921b79b1175dcb211336aaf84dfd8edcf19c07698934d4fe70e5eR25-R26

```suggestion
See if you can inspect the database. Username: `postgres`. Password: `postgres`.
```

Reply:
Good catch! I used the Java file but did not check all texts